.github/workflows/ci.yml

```diff
@@ -25,9 +25,21 @@ jobs:
           mix local.rebar --force
           mix local.hex --force
           mix deps.get
-      - name: Compile Code
-        run: mix compile --warnings-as-errors
       - name: Ensure Code is Formatted
         run: mix format --check-formatted
+      - name: Compile for Tests
+        run: MIX_ENV=test mix compile --warnings-as-errors
```

Comment:
We only compile for tests because the release itself will compile for `prod`. This ensures that `mix test` doesn't recompile code